### ISSUE TITLE
docs: use container images

### ISSUE THREAD

**Issue by mudler** [masa-finance/masa-bittensor#139]
As part of https://github.com/masa-finance/masa-bittensor/issues/109 we have now container images published by the pipelines.

We want now to be consistent and have documentation explaining how to use the images, and use these in place of having to setup the environment manually (via `git clone`, `pip` and running `make` commands).

**Comment by mudler** [comment]
Blocked by https://github.com/masa-finance/roadmap/issues/53

**Comment by teslashibe** [comment]
This is superseded by https://github.com/orgs/masa-finance/projects/7/views/1?pane=issue&itemId=70955991&issue=masa-finance%7Croadmap%7C53 this is now closed

**Comment by teslashibe** [comment]
This replaces https://github.com/orgs/masa-finance/projects/7/views/1?pane=issue&itemId=70955991&issue=masa-finance%7Croadmap%7C53

https://github.com/orgs/masa-finance/projects/7/views/1?pane=issue&itemId=70955991&issue=masa-finance%7Croadmap%7C53 is now closed
